src/compas/utilities/functions.py

```diff
@@ -19,6 +22,58 @@ def fibonacci(n, memo={}):
         memo[n] = fibonacci(n - 2, memo) + fibonacci(n - 1, memo)
     return memo[n]
 
+def avrg(list):
```

Comment:
I would suggest sticking to standard naming for these functions, in particular, taking from `numpy` stats functions, this would be `average`.

Reply:
agreed, and in any case words should not be abbreviated. what sounds like a meaningful abbreviation to you might not to someone else...

Reply:
full names now: average, variance, standard_deviation